### PR TITLE
feat(navbar): make navbar scroll sensitive

### LINKED ISSUE #1100: Feature: Make navbar scroll sensitive
### Description

I want to make the navbar such that when we scroll down it hides and when we scroll up it shows up again.
You can take demo of the feature from this website: https://blogzen-outerbase.netlify.app
Please assign this issue to me.

### Screenshots

_No response_

### 🥦 Browser

Google Chrome

### Checklist ✅

- [X] I checked and didn't find similar [issue](https://github.com/MilanCommunity/Milan/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/MilanCommunity/Milan/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (blank for no)

### PR DESCRIPTION
**Pull request by Palakkgoyal**
closes #1100 

### 👷🏻 Changes made  
Did changes in the navbar such that when we scroll down it does not appear and when we scroll up it become visible.

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -144,3 +149,18 @@ const Navbar = () => {
...
+    navRef.current.style.top =
+      scrollTop > lastScrollTop.current ? "-80px" : "0px";
+    lastScrollTop.current = scrollTop;
+  }
+
+  useEffect(() => {
+    window.addEventListener("scroll", handleScroll);
+    return () => window.removeEventListener("scroll", handleScroll);
+  }, []);
+}
```
Comment (tamalCodes) [inline src/components/Navbar/Navbar.jsx:166]: If this is another function which is then imported , I would recommend to make a file with a meaningful name and move it to the utils folder

File: src/components/Navbar/Navbar.jsx
Comment (tamalCodes) [inline src/components/Navbar/Navbar.jsx:1]: The changes work fine, however, would it be possible to make the transition smoother?

**Review by tamalCodes — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Navbar.jsx
Comment (Palakkgoyal) [reply src/components/Navbar/Navbar.jsx:1]: Yes, I think I can make it smoother

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (Palakkgoyal) [reply src/components/Navbar/Navbar.jsx:166]: Okay

**Comment by tamalCodes** [comment]
# Let's catch up 🙋🏻‍♂️
Hi there, it has been a while since we last had a conversation so I wanted to know if you are still working on this. Please drop a reply shortly so that we know you are active, feel free to convey any problem you face in the comments, or mail me at milanhub.org@gmail.com

**Comment by tamalCodes** [comment]
# Notice 🚩

Hi there, unfortunately, I will have to close this Issue/PR, as this issue/PR has been inactive for quite a while and there's no work going on for the same.

There might be more Issues/PR that are waiting for this to be completed - so it is better to pace things up. I generally spend/wait for 3 weeks before marking any issue as stale and closing it.

If you think that this is a mistake, please drop an email at milanhub.org@gmail.com, and we will get back to you.

Regards, 
Team Milan.

**Pull request closed without merging**